www/klaj01/SP2/js/javascript.js

```diff
@@ -182,10 +191,27 @@ function render() {
   drawArc(ball.x, ball.y, ball.radius, ball.color);
 }
 
+// animation will pause when paused==true
+var paused = false;
+
+// testing, a rotation angle for the animated rect
+var angle = 0;
+
 let framePerSecond = 60;
 
 let loop = setInterval(game, 1200 / framePerSecond);
 
+function fbshareCurrentPage() {
```

Comment:
Můžu tímhle stylem sdílet score na twitteru ? (Ještě přidám facebook a ikonky skryju při začátku hry a zobrazim po skončení)

Reply:
tohle neni AJAX, takze to neplati